keeper/src/main/scala/zio/keeper/membership/swim/Message.scala

```diff
@@ -0,0 +1,54 @@
+package zio.keeper.membership.swim
+
+import zio.duration.Duration
+import zio.keeper.membership.NodeAddress
+import zio.keeper.membership.swim.Message.{ NoResponse, WithTimeout }
+import zio.{ IO, ZIO, keeper }
+
+sealed trait Message[+A] {
+  self =>
+
+  final def transformM[B](fn: A => IO[keeper.Error, B]): IO[keeper.Error, Message[B]] =
```

Comment:
this could be fully polymorphic in the error

Reply:
because of action in ```WithTimeout``` it cannot be